fetch.bs

```diff
@@ -6631,6 +6705,64 @@ agent's <a>CORS-preflight cache</a> for which there is a <a>cache entry match</a
 </ol>
 </div>
 
+<h3 id=deferred-fetching>Deferred fetching</h3>
+
+<p>Deferred fetches allow callers to request that a fetch is invoked at the latest possible moment,
+i.e. when a <a>fetch group</a> is <a for="fetch group">terminated</a>, or after a timeout after it
+is <a for="fetch group">deactivated</a>.
+
+<div algorithm="request-a-deferred-fetch">
+<p>To <dfn>request a deferred fetch</dfn> given a
+<a for=/>request</a> <var>request</var> and a null-or-{{DOMHighResTimeStamp}}
+<var>inactivityDeferredDelay</var> (default null):
+
+<ol>
+ <li><p>Assert: <var>request</var>'s <a for=request>client</a> is an
+ <a>environment settings object</a>.
+
+ <li>Let <var>totalScheduledDeferredBytesForOrigin</var> be 0.
+
+ <li>
+  <p>If <var>request</var>'s <a for=request>body</a> is not null then:
+
+  <ol>
+   <li><p>If <var>request</var>'s
+   <a for=request>body</a>'s <a for=body>length</a> is null, then throw a {{TypeError}}.
+
+   <li><p>Set <var>totalScheduledDeferredBytesForOrigin</var> to <var>request</var>'s
```

Comment:
```
Set totalScheduledDeferredBytesForOrigin to request's body length.
```

Could the step to throw QuotaError below also be run here to allow early termination? This single request may be already exceeding the 1KB quota.

```
If  totalScheduledDeferredBytesForOrigin  is greater than 64 kilobytes, then throw a QuotaExceededError.
```

Reply:
You could do this in the implementation, it won't change anything behavior-wise.